docs/JSON_format.md

```diff
@@ -482,6 +482,14 @@ in the enclosing action.
 must both resolve to integral values, which we call respectively `cond` and
 `offset`. If `cond` is 0, the primitive behaves exactly like `_jump` and we jump
 to the primitive call at index `offset`; otherwise the primitive is a no-op.
+- `assume` and `assert`: these 2 statements are used in P4 for formal program
+verification. They both take a single parameter (a boolean expression) and they
```

Comment:
The documentation changes look good to me.

I just wanted to mention that given the run-time behavior, they are not only for formal program verification, but also test-based verification, too.